### ISSUE TITLE
RSolr.escape doesn't work for dash

### ISSUE THREAD

**Issue by jcoyne** [rsolr/rsolr#71]
When the parameter is `"aptrust-test:40110"` it is escaped as `"aptrust\\-test\\:40110"`, which causes an error: 
```
'error'=>{'msg'=>'org.apache.solr.search.SyntaxError: Cannot parse \'my_field:aptrust\\-test\\:40110}
```

**Comment by ndushay** [comment]
isn't this missing the end quote?  \'my_field:aptrust\\-test\\:40110    

also, better solr escaping coming ...

**Comment by ndushay** [comment]
@jcoyne may this be closed in favor of #98 ?

**Comment by ndushay** [comment]
ize closing this.
